ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-stable-4.13-loaded-upgrade-from-stable-4.12.yaml

```diff
@@ -0,0 +1,85 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-loaded-upgrade-412to413
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  intermediate:
+    release:
+      channel: candidate
+      version: "4.12"
+  latest:
+    release:
+      channel: candidate
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: perfscale-rosa-multiaz-6nodes-stage-loaded-upgrade412
+  cron: 0 8 * * 3
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: candidate
+      GC: "false"
+      IF_MINOR_Z_UPGRADE: "true"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: 4.12.26
+      REGION: us-east-2
+      ROSA_LOGIN_ENV: staging
+    test:
+    - ref: ipi-install-rbac
+    - ref: openshift-cluster-bot-rbac
+    - as: set-intermediate-releases
+      commands: echo "${RELEASE_IMAGE_INTERMEDIATE}" > ${SHARED_DIR}/perfscale-override-upgrade
+        && cat ${SHARED_DIR}/perfscale-override-upgrade
+      dependencies:
+      - env: RELEASE_IMAGE_INTERMEDIATE
+        name: release:intermediate
+      from: cli
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    - chain: openshift-qe-cluster-density-v2
+    - chain: openshift-qe-rosa-loaded-upgrade-and-postcheck
+    workflow: rosa-aws-sts
+  timeout: 8h0m0s
+- as: perfscale-rosa-multiaz-24nodes-stage-loaded-upgrade413
+  cron: 0 8 * * 3
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: candidate
+      COMPUTE_MACHINE_TYPE: m5.2xlarge
+      ENABLE_AUTOSCALING: "false"
+      GC: "false"
+      IF_MINOR_Z_UPGRADE: "false"
+      MULTI_AZ: "true"
+      OPENSHIFT_VERSION: "4.12"
+      READY_WAIT_TIMEOUT: 30m
+      REGION: us-west-2
+      REPLICAS: "6"
```

Comment:
Should be 24. And the above 'perfscale-rosa-multiaz-6nodes-stage-loaded-upgrade412' should have REPLICAS: "6".

Reply:
Fixed in new code, thanks1